src/components/DynamicWidgets.js

```diff
@@ -65,6 +69,7 @@ export default {
     });
 
     // by default, render everything, but hidden so that the routing doesn't disappear
+    // TODO: make sure fallback component gets rendered here too, but how do we know which attributes?
```

Comment:
I tried to find a similar solution with rendering/searching an extra time like in React InstantSearch, but couldn't find it :(

Reply:
What are the fundamental differences between CSR and SSR in the Dynamic widget?